tutorials/algorithms/06_grover.ipynb

```diff
@@ -292,12 +291,12 @@
       "text/plain": [
-       "<Figure size 267.197x204.68 with 1 Axes>"
+       "<Figure size 371.107x284.278 with 1 Axes>"
       ]
      },
-     "execution_count": 6,
+     "execution_count": 46,
      "metadata": {},
      "output_type": "execute_result"
     }
```

Comment:
I believe here the oracle can be simplified to only a [CCZGate](https://qiskit.org/documentation/stubs/qiskit.circuit.library.CCZGate.html#qiskit.circuit.library.CCZGate) with the [ccz method](https://qiskit.org/documentation/stubs/qiskit.circuit.QuantumCircuit.ccz.html).
```
oracle = QuantumCircuit(3)
oracle.ccz(0,1,2)
```